src/main/java/com/google/api/codegen/viewmodel/OptionalArrayMethodView.java

```diff
@@ -57,6 +57,8 @@
 
   public abstract boolean isLongrunningOperation();
 
+  public abstract boolean hasLongrunningReturnValue();
```

Comment:
It's possible, but I don't think so necessarily. Longrunning operations are now supported without them, so until we find additional features that need to be added, there won't be more changes